### PR TITLE
feat: Complete notification-template docs

### PR DESCRIPTION
**Pull request by codingmickey**
Completes all the tasks of novuhq/novu#1320

### CODE REVIEW

**Review by florian-lefebvre — CHANGES_REQUESTED** [review]
Please rename :
- getAll.md to get-all.md
- getOne.md to get-one.md
- updateStatus.md to update-status.md

File: content/code-examples/notification-templates/getOne.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+title: Novu get single notification template
+type: snippets
+---
+
+```javascript label=Node.js
+import { Novu } from '@novu/node';
+
+export const novu = new Novu('<REPLACE_WITH_API_KEY_FROM_ADMIN_PANEL>');
+
+await novu.notificationTemplates.create("templateId");
```
Comment (nilesh-maurya) [inline content/code-examples/notification-templates/getOne.md]: this will be  `await novu.notificationTemplates.getOne("templateId");`

Comment (codingmickey) [reply content/code-examples/notification-templates/getOne.md]: Yes, my bad.. will change

**Review by florian-lefebvre — APPROVED** [review]

**Comment by florian-lefebvre** [comment]
cc @scopsy

**Review by scopsy — APPROVED** [review]
Looks great!

**Pull request merged**